src/components/DevBar/DevBar.tsx

```diff
@@ -0,0 +1,37 @@
+import { Accordion, Button, Card, Icon } from "@canonical/react-components";
+import classNames from "classnames";
+
+import useLocalStorage from "hooks/useLocalStorage";
+
+import { useWidgets } from "./widgets";
+
+const MINIMISED_KEY = "DEV_minimised";
+
+export default function DevBar() {
+  const [minimised, setMinimised] = useLocalStorage(MINIMISED_KEY, false);
+  const sections = useWidgets();
+
+  if (!import.meta.env.DEV) {
+    console.error("`<DevBar />` rendered in non-dev mode");
+    return null;
+  }
+
+  return (
+    <div className="dev-bar">
+      {minimised && (
```

Comment:
I prefer to use a ternary for conditionally displaying jsx, otherwise falsy values sometimes get display on the page (e.g. "0"). Likely the local storage won't ever be set to something like `0` but I find it good practice to always use a ternary and then you don't have to consider what the values might be.